### PR TITLE
Revert "Revert "Db setup""

### PR DESCRIPTION
**Pull request by sibsmc**
Reverts RailsGirlsCPH/mentor-mentee-platform#59 back to square one.

### CODE REVIEW

**Comment by Insti** [comment]
This PR does a lot of things.
It would probably be a good idea to split it up into many smaller logically related PRs.

File: server/app/controllers/meetingfreqs_controller.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
+class MeetingfreqsController < ApplicationController
```
Comment (hcarreras) [inline server/app/controllers/meetingfreqs_controller.rb]: I would recommend not to shorten words. It's just a place to create misunderstandings.

**Comment by Insti** [comment]
(+1 on @hcarreras awesome and helpful review, thanks :heart:)

**Comment by sibsmc** [comment]
Thanks for the feedback, I'll close this as you suggest and start making smaller requests.

**Pull request closed without merging**